modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/test/ReverseProxyTest.java

```diff
@@ -36,6 +36,28 @@ public class ReverseProxyTest {
 
 	@Test
 	public void testActionsHrefValues() throws Exception {
+		_testActionsHrefValues("http://[0:0:0:0:0:0:0:1]/", 80, "http");
```

Comment:
Not sure if we should create or update any other test, as Reverse Proxy is not needed for testing our purposes, so it is a bit weird to test it in here

Reply:
This test was originally created to test exactly this (even if the name is confusing).